html_test/html_test.go

```diff
@@ -0,0 +1,37 @@
+package html_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"code.sajari.com/docconv"
+)
+
+func TestConvertHTML(t *testing.T) {
```

Comment:
Even if we aren't able to accept the package global change above, thank you for writing this test. We'd love to get this merged. Per my comment above I don't think we can change the package global variable. However, in the first instance could you change the PR to remove the package global change, but keep this test adapted to verify the package's existing behavior? Thanks!

Reply:
Are the tests to your satisfaction or would you like me to make more changes?

Reply:
Hi @remLse apologies for the delay in reviewing this. Will take a look as soon as possible. Thanks for your patience.